examples/AlwaysOffTraceExample.php

```diff
@@ -0,0 +1,28 @@
+<?php
+require __DIR__.'/../vendor/autoload.php';
+use OpenTelemetry\Tracing\Builder;
+use OpenTelemetry\Tracing\SpanContext;
+use OpenTelemetry\Tracing\Sampler\AlwaysOffSampler;
+$sampler = AlwaysOffSampler::shouldSample();
+if($sampler) {
+    $spanContext = SpanContext::generate(); // or extract from headers
+    $tracer = Builder::create()->setSpanContext($spanContext)->getTracer();
```

Comment:
Was the builder removed in another PR?

Reply:
Yup; this gets removed in the next PR:
https://github.com/open-telemetry/opentelemetry-php/pull/31